scientific/src/commonTest/kotlin/ScientificValueTest.kt

```diff
@@ -52,5 +55,9 @@ class ScientificValueTest {
         assertEquals(0.5(Hertz), 1.0 / 2(Second))
         assertEquals(3(Kilogram), 9(Kilogram) / 30(Hectogram))
         assertEquals(3(Kilogram), 9(Kilogram) / 3)
+
+        assertFailsWith<ScientificConverterException> { 1(Kilogram) / 0(Kilogram) }
+        assertFailsWith<ScientificConverterException> { 1(Kilogram) / 0 }
+        assertFailsWith<ScientificConverterException> { 0(Second).frequency() }
```

Comment:
What's wrong with 0Hz? :)

Reply:
Frequency is 1/time, so at 0 seconds, you get a Divide by Zero exception on Frequency. Or to put it another way, if the interval between events is 0 seconds, you get an infinite amount of events per second.

Reply:
0(Hz) it still possible